compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -165,7 +167,7 @@ impl CodegenCx {
 }
 
 pub struct CraneliftCodegenBackend {
-    pub config: RefCell<Option<BackendConfig>>,
+    pub config: RwLock<Option<BackendConfig>>,
```

Comment:
This can be a Mutex too. Or probably even a `OnceLock`.

Reply:
I've changed it to `OnceLock`.